src/v/iceberg/rest_client/catalog_client.cc

```diff
@@ -288,6 +350,10 @@ ss::future<expected<create_namespace_response>>
 catalog_client::create_namespace(
   create_namespace_request req, retry_chain_node& rtc) {
     auto gh = _gate.hold();
+    auto config_result = co_await maybe_configure(rtc);
```

Comment:
i think i only saw trace logging in maybe_configure i wonder if the first successful configure (i'm assuming that happens just once) should announce at info level what happened, or else we see that information some where else?

Reply:
You found em